htdocs/expensereport/class/paymentexpensereport.class.php

```diff
@@ -646,7 +646,7 @@ public function getNomUrl($withpicto = 0, $maxlen = 0)
 			$label .= '<br><b>'.$langs->trans('Ref').':</b> '.$this->ref;
 		}
 		if (!empty($this->datep)) {
-			$label .= '<br><b>'.$langs->trans('Date').':</b> '.dol_print_date($this->datep, 'dayhour');
+			$label .= '<br><b>'.$langs->trans('Date').':</b> '.dol_print_date($this->db->jdate($this->datep), 'dayhour');
```

Comment:
$this->datep is a memory variable so content must already be formated and the db->jdate shoul dbe useless.
The db->jdate is probably missing in the function that is setting the $this->datep